lib/slide_hub/cloud/engine/azure.rb

```diff
@@ -153,14 +155,18 @@ def self.generate_sas_url(blob_name)
           uri = bs.generate_uri Addressable::URI.escape("#{@config.bucket_name}/#{blob_name}"), {}
           uri.scheme = 'https'
 
-          signer = ::Azure::Contrib::Auth::SharedAccessSignature.new(uri, {
-            resource:    'b',
+          signer = ::Azure::Storage::Core::Auth::SharedAccessSignature.new(
+            @config.azure_storage_account_name,
+            @config.azure_storage_access_key)
```

Comment:
__[RuboCop]__

Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.


<!-- Comment by [SideCI](https://sideci.com) -->